web/themes/custom/votegov/templates/navigation/menu--main--votegov-mainnavigation.html.twig

```diff
@@ -35,6 +35,10 @@ We call a macro which calls itself to render the full tree.
   {{ menus.menu_links(items, attributes, 0, 'basic-mega-nav-section-') }}
   <div class="usa-nav__secondary">
     {{ drupal_entity('block_content', 2) }}
+     {% if language == 'en' %}
+        <button class="usa-button" id="vote-touchpoints--button" title="{{ 'How can we improve vote.gov?' | t }}" data-test="touchpointsBtn">{{ 'How can we improve vote.gov?' | t }}</button>
+        <script src="https://touchpoints.app.cloud.gov/touchpoints/c169d3b2.js" async></script>
```

Comment:
Is this script tag loaded twice on the page?

Reply:
This is now only loaded once on the page.